c10/cuda/CUDAMiscFunctions.cpp

```diff
@@ -12,7 +12,7 @@ const char* get_cuda_check_suffix() noexcept {
   } else {
     return "\nCUDA kernel errors might be asynchronously reported at some"
            " other API call, so the stacktrace below might be incorrect."
-           "\nFor debugging consider passing CUDA_LAUNCH_BLOCKING=1.";
```

Comment:
This is the end of a sentence? :p

Reply:
Technically true, but it makes it more annoying to copy-paste. If you want, I can add a space and then a period :)

Reply:
Is it? You can't double click it anyways because it stops at the sign...
I think adding extra quotes might be best? This is the solution that is the least likely that someone else will send a PR reverting your change haha

Reply:
the double clicking works for me in vscode terminal :^)